src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,34 @@
-export const TodoInfo = () => {};
+import cn from 'classnames';
+
+import { Todo } from '../../types';
+import { UserInfo } from '../UserInfo';
+
+type Props = {
+  todo: Todo,
+};
+
+export const TodoInfo: React.FC<Props> = (props) => {
+  const { todo } = props;
```

Comment:
Yo can simplify this:
```suggestion
export const TodoInfo: React.FC<Props> = ({ todo }) => {
```